lldb/cmake/modules/FindPythonAndSwig.cmake

```diff
@@ -64,3 +64,10 @@ else()
                                       Python3_EXECUTABLE
                                       LLDB_ENABLE_SWIG)
 endif()
+
+set(LLDB_RECOMMENDED_PYTHON "3.8")
+if(Python3_LIBRARIES AND Python3_INCLUDE_DIRS AND Python3_EXECUTABLE AND LLDB_ENABLE_SWIG
```

Comment:
Can this check `PYTHONANDSWIG_FOUND` to encompass everything on this line?

Reply:
Done.